pyproject.toml

```
@@ -31,7 +31,7 @@ dependencies = [
   "googleapis-common-protos >= 1.56.4, < 2",
   "grpcio >= 1.60.0, < 2",
   "frequenz-channels >= 1.0.0, < 2",
-  "frequenz-client-base >= 0.3.0, < 0.4",
+  "frequenz-client-base >=0.5",
```

I think the dependency should now include the grpc dependency, like so

```suggestion
  "frequenz-client-base[grpcio] >= 0.5.0, < 0.6.0",  
```